templates/ruff.toml

```diff
@@ -105,8 +105,10 @@ ignore = [
     "FIX",  # TODOs, etc: ~400 errors
 ]
 
+{{- if -eq .REPO_NAME "chia-blockchain"}}
 [lint.per-file-ignores]
 "chia/_tests/*" = ["S106"]
+{{- end}}
```

Comment:
I would maybe just name this variable based on what its doing (`RUFF_BLOCKCHAIN_ONLY_INCLUDES` or something?). None of the variables here are going to be automatically inherited when this runs, and have to be set in here: https://github.com/Chia-Network/chia-blockchain/blob/main/.repo-content-updater.yml#L1

So might as well have it be descriptive?